Added new kb article display-week-number-scheduler-header-maui (#988)
telerik/maui-docs: knowledge-base/display-week-number-scheduler-header-maui.md

---

> title: Displaying Week Number in Scheduler HeaderTextFormat in .NET MAUI

Reviewer: I suggest reducing the length as .NET MAUI is added by default during the build.
```suggestion
title: Displaying Week Numbers in the Scheduler's HeaderTextFormat
```

description: Learn how to show week numbers in the RadScheduler for .NET MAUI by binding the HeaderTextFormat property.
type: how-to
page_title: How to Display Week Numbers in Scheduler Header in .NET MAUI
slug: display-week-number-scheduler-header-maui
tags: scheduler, .net maui, headertextformat, binding, week number
res_type: kb
---

## Environment

| Version | Product | Author | 
| --- | --- | ---- | 
| 7.1.0 | Telerik UI for .NET MAUI Scheduler | [Dobrinka Yordanova](https://www.telerik.com/blogs/author/dobrinka-yordanova) | 

## Description

I need to display the week number in the `HeaderTextFormat` of the [Scheduler]({%slug header-settings%}) for .NET MAUI. How can I achieve this? Is it possible to use a bindable property for customization?

This KB article also answers the following questions:

> - How to bind a property to the Scheduler's HeaderTextFormat in .NET MAUI?

Reviewer: ```suggestion
- How to bind a property to the Scheduler's `HeaderTextFormat` in .NET MAUI?
```

- What is the method to customize the Scheduler's header format to include week numbers?

> - Can I use data binding to modify the HeaderTextFormat of the Scheduler control?

Reviewer: ```suggestion
- Can I use data binding to modify the `HeaderTextFormat` of the Scheduler control?
```

## Solution

> To display the week number in the `HeaderTextFormat` of the RadScheduler, you can utilize data binding for customization. The `HeaderTextFormat` property is bindable, allowing you to format the date in the header as required.

Reviewer: ```suggestion
To display the week number in the `HeaderTextFormat` of the `RadScheduler`, you can utilize data binding for customization. The `HeaderTextFormat` property is bindable, allowing you to format the date in the header as required.
```
